src/IO.Dyspatch.Tests/IntegrationTest.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Xunit;
+using IO.Dyspatch.Api;
+using IO.Dyspatch.Client;
+using IO.Dyspatch.Model;
+
+namespace IO.Dyspatch.Tests
+{
+    public class IntegrationTest
+    {
+        DraftsApi drafts;
+        TemplatesApi templates;
+
+        public IntegrationTest()
+        {
+            Configuration config = new Configuration();
+            config.ApiKey.Add(
+                "Authorization",
+                Environment.GetEnvironmentVariable("DYSPATCH_API_KEY")
+            );
+            config.ApiKeyPrefix.Add("Authorization", "Bearer");
+
+            drafts = new DraftsApi(config);
+            templates = new TemplatesApi(config);
+        }
+
+
+        [Fact]
+        public void GetTemplates()
+        {
+            var accept = "application/vnd.dyspatch.2019.10+json";
+            var id = "tem_01de5teh6k59kya8q92mb01qzq";
+            var lang = "handlebars";
+
+            templates.GetTemplates(accept, "");
+            templates.GetTemplateById(id, lang, accept);
```

Comment:
should there be some sort of assertion at least that we don't a 200 back?

Reply:
I believe the client library will throw an exception if it fails, but we could add an additional assertion. What would you suggest?

Reply:
Ok you are right, for any status code >= 400 it throws an Api Exception that works for me for now.